content/backstage/plugins/travis-ci.md

```diff
@@ -33,10 +33,15 @@ gettingStarted:
           target: https://api.travis-ci.com
           changeOrigin: true
           headers:
-            Authorization:
-              $secret:
-                env: TRAVISCI_AUTH_TOKEN
+            Authorization:${TRAVISCI_AUTH_TOKEN}
```

Comment:
Does it matter that there is no space between the `Authorization:` and the `${TRAVISCI_AUTH_TOKEN}`.